numba/np/arraymath.py

```diff
@@ -4982,20 +4981,23 @@ def np_in1d_impl(ar1, ar2, assume_unique=False, invert=False, kind="sort"):
 def jit_np_isin(element, test_elements, assume_unique=False, invert=False,
                 kind="sort"):
```

Comment:
The default value for `kind` should be `None`, not `"sort"`.

The following should work on your branch:
```python
import numpy as np
from numba import njit

@njit
def foo():
    return np.isin(3, 4, kind=None)

print(foo())
```

Reply:
The reason I've done it like I did is because I didn't implement the numpy ``kind="table"`` method. (The output is equivalent, but I suppose the performance is higher in certain cases with the table method (I haven't tested)). When ``kind=None``, the numpy code will use the table method if certain conditions are met, otherwise the sorting method, so we don't support that either -- strictly, we only support ``kind="sort"``. So I thought ``np.isin(3, 4, kind=None)`` *should* fail.

If you think it should work I'm happy to change it, or otherwise we just say we don't support the "kind" argument and leave it at that. Or, I could even implement the "table" method, it isn't that much more complicated than the sort method, and then we fully support the "kind" argument.

Reply:
Okay. I think it is best to match as close as possible the NumPy signature but keep a check internally for `kind == "sort"`.